docs/content/rules/sort-imports.mdx

```diff
@@ -201,6 +201,12 @@ Specifies a maximum line length for sorting imports. When the line length exceed
 
 This option is only available when the type is set to `'line-length'`.
 
+### tsConfigRootDir
```

Comment:
`tsconfigRootDir` (with a small `c`) would be more consistent with the already existing eslint config option: https://typescript-eslint.io/getting-started/typed-linting/

Reply:
@OlivierZal Changed in e81054b79bd07ce2977806ddd39f0590592fbf9d.